### ISSUE TITLE
[JENKINS-71499] WCAG Operable:2.1 - Keyboard operation and navigation - Result 7. The focus order is not logical in some places

### ISSUE THREAD

**Issue by jenkins-infra-bot** [timja-org/jenkins-gh-issues-poc-11-14#7029]
<p>On the "Settings" page in the user profile and on the "Configure" page, help icons are offered for many input fields, which provide more detailed information on the fields.</p>

<p>These icons are at the top of the focus order and are jumped to one after the other, regardless of the associated input fields when navigating with the keyboard.</p>

<p>This means that there is no connection to the corresponding form fields and blind users in particular cannot establish a connection between the icons and the form fields.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=cristinap">cristinap</a>, imported from: <a href="https://issues.jenkins.io/browse/JENKINS-71499" target="_blank">WCAG Operable:2.1 - Keyboard operation and navigation - Result 7. The focus order is not logical in some places</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-14
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>On the "Settings" page in the user profile and on the "Configure" page, help icons are offered for many input fields, which provide more detailed information on the fields.</p>

<p>These icons are at the top of the focus order and are jumped to one after the other, regardless of the associated input fields when navigating with the keyboard.</p>

<p>This means that there is no connection to the corresponding form fields and blind users in particular cannot establish a connection between the icons and the form fields.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
<i>[Epic: <a href="https://github.com/timja-org/jenkins-gh-issues-poc-11-14/issues?q=in%3Atitle%20JENKINS-71153%20label%3Aepic">JENKINS-71153</a>]</i>
